### DISCUSSION TITLE
Remove GitHub "edit this page" link for auto-generated pages

### DISCUSSION THREAD

**Discussion by ecomodeller** [machow/quartodoc#337] [Q&A] [answered]
Quarto has a nice feature to add a helpful "edit this page" link
https://quarto.org/docs/websites/website-navigation.html#github-links
![image](https://github.com/machow/quartodoc/assets/614215/49de4f9e-b147-4330-9373-feeb031fd883)

This lowers the barrier for people to contribute to your documentation.

But for the Reference pages (auto-generated by quartodoc), I would like to remove those links, since editing that page would be useless.

According to the Quarto docs I could add 
```yaml
---
repo-actions: false
---
```
to the pages where I want to remove these links, but how do I add this to the quartodoc generated pages?

**Comment by machow** [comment, marked as answer]
Hey--I'm so sorry for the wait. Would this work?

1. Put `_metadata.yml` in the folder where your API pages were generated (e.g. `api/_metadata.yml`)
2. Add the `repo-actions: false` there

https://quarto.org/docs/projects/quarto-projects.html#directory-metadata
